modules/core/shared/src/main/scala/gem/math/MagnitudeValue.scala

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.math
+
+import cats.{ Order, Show }
+import cats.instances.int._
+
+/**
+ * Exact magnitude value represented as an int with the original value scaled up
+ * @param value This magnitude integral value
+ */
+final case class MagnitudeValue(val value: Int) extends Product with Serializable
```

Comment:
Since this is 100x the actual magnitude maybe it shouldn't be visible, or at least not so easy to confuse with the actual magnitude.  Maybe call it `scaledValue` or something and add a `toDouble: Double` method that gets the actual magnitude?

Reply:
👍

Reply:
`scaledValue` is used on test. Would it be ok to declare it `private` for the `gem` package?